frappe/custom/doctype/customize_form/customize_form.py

```diff
@@ -356,7 +356,7 @@ def update_in_custom_field(self, df, i):
 
 	def delete_custom_fields(self):
 		meta = frappe.get_meta(self.doc_type)
-		fields_to_remove = (set([df.fieldname for df in meta.get("fields")])
+		fields_to_remove = (set(df.fieldname for df in meta.get("fields"))
```

Comment:
Might as well go all the way

```suggestion
		fields_to_remove = {df.fieldname for df in meta.fields}
```

Reply:
Yeah, will change this and similar instances.